sdk/go/common/workspace/project_test.go

```diff
@@ -900,10 +898,8 @@ func TestProjectLoadYAML(t *testing.T) {
 	_, err = loadProjectFromText(t, "name: project\nruntime: 4")
 	// These can vary in order, so contains not equals check
 	expected := []string{
-		"3 errors occurred:",
-		"* #/runtime: oneOf failed",
-		"* #/runtime: expected string, but got number",
-		"* #/runtime: expected object, but got number"}
+		"1 error occurred:",
+		"* #/runtime: expected string, but got number; expected object, but got number"}
```

Comment:
I'd like to see more tests for this before merging.
We're currently missing errors for various config "mistakes" and we should write and merge those tests first, and then rebase these changes and see how they compare.
I'm not confident just dropping all intermediate messages will always be clear.

Reply:
@Frassle could you give a few examples of the type of config mistakes so I can check against what we currently have?

Reply:
Sure stuff like having:
* "value" + one of the other schema options.
* type set to "array" but no "items" block
* type not set to array but having an items block
* type set to array but items not being a valid schema block

Non config related:
* setting "backend" to a string directly
* mis-spelling "default" (e.g. "dflt") in a template config section
* Setting the plugins.providers array as a map
* Setting runtime.options to a string